CustomApps/lyrics-plus/OptionsMenu.js

```diff
@@ -134,6 +136,8 @@ const TranslationMenu = react.memo(({ showTranslationButton, translatorLoaded, i
 								CONFIG.visual[name] = value;
 								localStorage.setItem(`${APP_NAME}:visual:${name}`, value);
 								lyricContainerUpdate && lyricContainerUpdate();
+								CONFIG.visual[name] ? Spicetify.showNotification("Translating...", false, 500) : null;
+								translator.includeExternal("inject");
```

Comment:
Could it be in a new function instead?
```javascript
translator.injectExternals();
```

Reply:
you mean this part as a separate fuction? then ofcourse. Do you want me to create a separate fuction to trigger that inject method instead of that if else thing?
![image](https://user-images.githubusercontent.com/75513645/215153472-b6e46130-c28b-4701-86dd-14576ed3a59b.png)

Reply:
That what would be the best I guess?
```javascript
injectExternals() {
	this.includeExternal(kuroshiroPath);
	this.includeExternal(kuromojiPath);
}
```